lib/security/waf.ts

```diff
@@ -110,7 +199,68 @@ export interface WafProps extends StackProps{
 
 export class JenkinsWAF {
   constructor(stack: Stack, props: WafProps) {
-    const waf = new WAF(stack, 'WAFv2');
+    // Create IP Sets for IPv4 and IPv6
+    const ipSetIpv4 = new CfnIPSet(stack, 'GitHubIpv4Set', {
+      name: 'GitHubIpv4Set',
```

Comment:
```suggestion
      name: 'AllowGitHubIpv4Set',
```

Reply:
This is just the IP set name and I don't think we should imply the action on these sets. We create the rule named `AllowGitHubIPv4`.